ext/autocomplete/style.css

```diff
@@ -0,0 +1,8 @@
+#Navigationleft .blockbody { overflow: visible; }
+
+.tagit { background: white !important; border: 1px solid grey !important; cursor: text; }
+.tagit-choice { cursor: initial; }
+input[name=search] ~ input[type=submit] { display: block !important; }
```

Comment:
Hadn't even considered cases where the search is on the same line. Shall fix that.